spec/spec_helper.rb

```diff
@@ -83,7 +83,7 @@
     ]
   end
 
-  config.include Devise::TestHelpers, type: :controller
+  config.include Devise::Test::ControllerHelpers, type: :controller
```

Comment:
This raises a warning. Originally pushed in https://github.com/coopdevs/timeoverflow/commit/75aa85d33524f2d9a55a27be9ee9e105b3ff93f1, and overridden by #333.